src/Dscp/CLI/Common.hs

```diff
@@ -39,6 +40,13 @@ dbPathParser = strOption $
     help "Path to database directory for witness node." <>
     value "witness-db"
 
+sqliteDbPathParser :: Parser FilePath
+sqliteDbPathParser = strOption $
+    long "db-path" <>
```

Comment:
The key is the same as for `dbPathParser` by the way, please don't copypaste that hard.